spec/support/features/idv_step_helper.rb

```diff
@@ -13,9 +13,9 @@ def self.included(base)
     end
   end
 
-  def start_idv_from_sp(sp = :oidc, facial_match_required: nil)
+  def start_idv_from_sp(sp = :oidc, acr_values: Saml::Idp::Constants::IAL_VERIFIED_ACR)
```

Comment:
should we document the type of `acr_values`? the reason I mention is that `acr_values` is plural so my guess from reading is that it would would be an array, but this value is a single string.
```suggestion
  # param acr_values [String] string with space-separated list of ACR values
  def start_idv_from_sp(sp = :oidc, acr_values: Saml::Idp::Constants::IAL_VERIFIED_ACR)
```

Reply:
So 2 things are happening:
1. OIDC calls the parameter `acr_values` and it MUST be a space-delimited string 
2. We've adopted OIDC's naming convention and now `acr_values` sometimes means an array and sometimes it means a string.

I think it is beyond the scope of this particular PR to fix that particular mixing of types in the naming conventions, but should be fixed in the future. I'll write up a ticket for it.

Reply:
I think the documentation for the expected type as suggested makes sense